src/main/java/com/robinpowered/sdk/model/Event.java

```diff
@@ -275,10 +275,11 @@ public String getMimeType() {
         private final DateTime startedAt;
         private final DateTime endedAt;
         private final boolean isAllDay;
-        private final List<Invitee.Invitation> invitees;
+        private final List<Invitable> invitees;
 
         public Booking(String ownerRef, String title, String description,
-                       DateTime startedAt, DateTime endedAt, boolean isAllDay, List<Invitee.Invitation> invitees) {
+                       DateTime startedAt, DateTime endedAt, boolean isAllDay,
+                       List<Invitable> invitees) {
```

Comment:
Yeah I know android's code style... I have intellij set up for 120 character line lengths. Let's just undo this for now and we'll establish a java style guide later.